pkg/charm/path.go

```diff
@@ -0,0 +1,37 @@
+package charm
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Path []*instance
```

Comment:
NIt: This isn't part of the `pkg/charm` interface, so don't export it.
```suggestion
type path []*instance
```